pages/documentation/guidance/accessibility.md

```diff
@@ -6,83 +6,167 @@ redirect_from:
-          <strong>show</strong>
-        </span>
-      </td>
-      <td data-title="Screen readers">
-        <span>
-          hide
-        </span>
-      </td>
-      <td data-title="Solution">
-        <span>
-          Use the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden"><code>aria-hidden="true"</code></a> attribute to "hide" content from screen readers while keeping it visually perceivable.
-        </span>
-      </td>
-    </tr>
-    <tr>
-      <td data-title="Visual browsers">
-        <span>
-          hide
-        </span>
-      </td>
-      <td data-title="Screen readers">
-        <span>
-          <strong>show</strong>
-        </span>
-      </td>
-      <td data-title="Solution">
-        <span>
-          Use the <code>usa-sr-only</code> class to show content to screen readers only. This removes the content from the visual flow of the document but retains its legibility by screen readers.
-        </span>
-      </td>
-    </tr>
-  </table>
-</div>
+When it comes to accessibility (sometimes shortened to "a11y" for the eleven characters between the letters "a" and "y" in the word "accessibility"), legal requirements are a critical, necessary starting point, but requirements are only the beginning. Accessibility is about real people who use our services — it's usability for any ability. Accessibility enables full participation, and everyone who works on government websites has a role to play in making federal resources accessible and inclusive. Incorporate accessibility from the start, design generously, and celebrate accessibility requirements as a set of design constraints that help us build products and services for those who need it most, and better products and services for all users.
+
+Today, when we think practically about accessibility in digital services, we think about the requirements of Section 508 of the Rehabilitation Act of 1973, amended in 1998 to require Federal agencies to make their electronic and information technology accessible to people with disabilities. And we think about the requirements of WCAG, or the Web Content Accessibility Guidelines, an evolving set of requirements, now in version 2.1. These requirements are related. Section 508 incorporates by reference the WCAG 2.0 Level AA success criteria.
+
+The goal of both these requirements is to provide content and services that are Perceivable, Operable, Understandable, and Robust:
+
+- **Perceivable:** Users must be able to perceive the information presented.
+- **Operable:** Users must be able to operate the interface.
+- **Understandable:** Users must be able to understand both how the user interface operates and the meaning of information presented.
+- **Robust:** Users must be able to access the content as technologies advance. As technologies like browsers and other assistive technology evolve, the content should remain accessible.
+
+## Common ability-based patterns the design system addresses
```

Comment:
I'm not sure if you're looking for prose feedback, but I had to read this line a few times, probably because it's in the passive voice, to understand it.
```## The design system addresses common ability-based patterns```
```## The common, ability-based patterns which the design system addresses```
```## What ability-based patterns does the design system address?```

Reply:
Rewrite this head1 to read more simply: "Common patterns USWDS addresses"